python/coinbase-agentkit/coinbase_agentkit/wallet_providers/eth_account_wallet_provider.py

```diff
@@ -9,16 +9,15 @@
 from web3.middleware import SignAndSendRawMiddlewareBuilder
 from web3.types import BlockIdentifier, ChecksumAddress, HexStr, TxParams
 
-from ..network import Network
+from ..network import NETWORK_ID_TO_CHAIN, NETWORK_ID_TO_CHAIN_ID, Network
 from .evm_wallet_provider import EvmWalletProvider
 
 
 class EthAccountWalletProviderConfig(BaseModel):
     """Configuration for EthAccountWalletProvider."""
 
     private_key: str
-    rpc_url: str
-    chain_id: int
+    network_id: str
```

Comment:
can we update this config to take an eth account instead of a private key so it is more flexible like the viem wallet provider?